driver/image_adjust_hue_driver.hpp

```diff
@@ -0,0 +1,389 @@
+    }
+    assert(input_vec.lengths[1] == 3);
+
+    auto strides = ComputeStrides(input_vec.lengths, inflags.GetValueInt("contiguous") == 1);
+
+    SetTensorNd(inputTensorDesc, input_vec.lengths, strides, data_type);
+    SetTensorNd(outputTensorDesc, input_vec.lengths, strides, data_type);
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustHueDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
+{
+    size_t in_size  = GetTensorSize(inputTensorDesc);
+    size_t out_size = GetTensorSize(outputTensorDesc);
+
+    uint32_t ctx = 0;
+
+    input_gpu  = std::unique_ptr<GPUMem>(new GPUMem(ctx, in_size, sizeof(Tgpu)));
+    output_gpu = std::unique_ptr<GPUMem>(new GPUMem(ctx, out_size, sizeof(Tgpu)));
+
+    in_host  = std::vector<Tgpu>(in_size, static_cast<Tgpu>(0));
+    out_host = std::vector<Tgpu>(out_size, static_cast<Tgpu>(0));
+    out_ref  = std::vector<Tref>(out_size, static_cast<Tref>(0));
+
+    for(auto i = 0; i < in_size; i++)
+    {
+        in_host[i] = static_cast<Tgpu>(prng::gen_A_to_B(0.0f, 1.0f));
+    }
+
+    if(input_gpu->ToGPU(GetStream(), in_host.data()) != miopenStatusSuccess)
+        std::cerr << "Error copying buffer to GPU with size " << input_gpu->GetSize() << std::endl;
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustHueDriver<Tgpu, Tref>::RunForwardGPU()
+{
+    float kernel_total_time = 0;
+    float kernel_first_time = 0;
+
+    Timer t;
+    START_TIME
+
+    for(int i = 0; i < inflags.GetValueInt("iter"); i++)
+    {
+        miopenImageAdjustHue(GetHandle(),
+                             inputTensorDesc,
+                             outputTensorDesc,
+                             input_gpu->GetMem(),
+                             output_gpu->GetMem(),
+                             hue);
+        float time = 0.0;
+        miopenGetKernelTime(GetHandle(), &time);
+        kernel_total_time += time;
+        if(i == 0)
+            kernel_first_time = time;
+    }
+
+    if(inflags.GetValueInt("time") == 1)
+    {
+        STOP_TIME
+        int iter = inflags.GetValueInt("iter");
+
+        if(WALL_CLOCK)
+            std::cout << "Image Adjust Hue Forward Time Elapsed: " << kernel_total_time << " ms"
+                      << std::endl;
+
+        float kernel_avg_time =
+            iter > 1 ? (kernel_total_time - kernel_first_time) / (iter - 1) : kernel_total_time;
+        std::cout << "Image Adjust Hue Kernel Time Elapsed: " << kernel_avg_time << " ms"
+                  << std::endl;
+    }
+
+    if(output_gpu->FromGPU(GetStream(), out_host.data()) != miopenStatusSuccess)
+    {
+        std::cerr << "Error copying buffer from GPU with size " << output_gpu->GetSize()
+                  << std::endl;
+        return -1;
+    }
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustHueDriver<Tgpu, Tref>::RunBackwardGPU()
+{
+    // Does not exist
```

Comment:
Please remove comment

Reply:
This is probably necessary in order to note that the backward functions for these do not exist, and thus running them will simply return.